Documentation/DE/Konfiguration/MultidomainEinrichten/Index.rst

```diff
@@ -0,0 +1,44 @@
+.. ==================================================
```

Comment:
Thanks for documenting this! Could you also please add the same to the English manual?

Reply:
I tried it...